addons/project/models/project.py

```diff
@@ -1334,6 +1362,9 @@ def _message_post_after_hook(self, message, msg_vals):
     def action_assign_to_me(self):
         self.write({'user_id': self.env.user.id})
 
+    def action_unassign_me(self):
```

Comment:
This action is not restrict to you only.

```suggestion
    def action_unassign(self):
```